recipes/dbus/1.x.x/conanfile.py

```diff
@@ -3,9 +3,10 @@
 
 from conan import ConanFile
 from conan.tools.files import apply_conandata_patches, copy, get, mkdir, rename, rmdir, save, rm
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.scm import Version
 from conans import CMake
-# TODO: Update to conan.tools.apple after 1.51.3
-from conans.tools import is_apple_os
 
 required_conan_version = ">=1.50.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.3"
```

The `is_apple_os` only works since 1.51.3: https://docs.conan.io/en/latest/changelog.html#id30